tests/plugins/emoji/manual/dropdown/smallrefactoring.md

```diff
@@ -0,0 +1,21 @@
+@bender-tags: 4.11.2, bug, emoji, 2592
```

Comment:
I'd rename this test.
Tests are supposed to check if code works, not if it looks nice, so refactoring doesn't sound like something that should be mentioned in test name. I understand that these cases are unrelated and it's hard to find a generic name that fits both, so maybe consider splitting them for separate test cases? It will be easier to give names to these cases as separate tests.

Also test steps from the first case can't be done on mobiles, so you should either add proper information in a description, or separate cases so the first can be ignored on mobiles.

Reply:
For sure the test name is really bad, _smallrefactoring_ tells you nothing about the test. Similarly to `thatOneThing.md` or `simplecase.md`. Always avoid names like that.

As mentioned by @engineering-this giving it a proper name would be tricky. Generic test will also be problematic to match this behavior. Idea for making two micro manual tests sounds really appealing.